rust/operator-binary/src/controller.rs

```diff
@@ -1637,7 +1648,7 @@ mod tests {
           name: simple-trino
         spec:
           image:
-            productVersion: "455"
+            productVersion: "469"
```

Comment:
```suggestion
            productVersion: "470"
```

Reply:
Done in dbf9409e08c5b82aff7d4724af1fa5b014e96fbf